gnovm/stdlibs/runtime/runtime.gno

```diff
@@ -0,0 +1,35 @@
+// Package runtime allows access to the GnoVM's runtime information.
+package runtime
+
+func NewReadOnlyBanker() chain.Banker {
+}
+
+func NewLimitedSendBanker() chain.Banker {
+}
+
+func NewSendBanker() chain.Banker {
+}
+
+func NewIssueBanker() chain.Banker {
+}
+
+func AssertOriginCall() {
+}
+
+func CallerRealm() chain.Realm {
```

Comment:
I am of a hot take that we should completely remove the possibility of a realm being an EOA. It's very confusing. Then, we could just have `Caller`, which can be a realm of an EOA/user.